### PR TITLE
Howard windows updates

### PR DESCRIPTION
**Pull request by hobu**
Catching up some windows build stuff to silence a few warnings and get it working on my 2019 build

### CODE REVIEW

File: filters/StatsFilter.hpp [hunk 1]
```diff
@@ -89,7 +89,7 @@ typedef std::vector<double> DataVector;
     {
         if (M2 == 0 || m_cnt <= 2 || !m_advanced)
             return 0.0;
-        double c(m_cnt);
```
Comment (abellgithub) [inline filters/StatsFilter.hpp]: We shouldn't have to do this.  What terrible warning is MS giving?

Comment (hobu) [reply filters/StatsFilter.hpp]: an implicit conversion warning.

Comment (abellgithub) [reply filters/StatsFilter.hpp]: Wonder if we should suppress.  That seems anal.

**Comment by abellgithub** [comment]
This is all fine, but I really hate such casts.  They confuse me -- why would I be doing this?  I'd rather suppress the compiler warning, but it's up to you.  Merge as is or just suppress the warning.  Up to you.

**Pull request merged**